openwisp_firmware_upgrader/upgraders/openwrt.py

```diff
@@ -90,32 +91,32 @@ def _test_checksum(self, image):
                     'Identical checksum found in the filesystem, '
                     'upgrade not needed.'
                 )
-                self.log(message)
+                self.log(_(message))
```

Comment:
Let's translate the string where it is defined instead for better code readability and keep it as it is


   
```suggestion
                 message = _(
                    'Firmware already upgraded previously. '
                    'Identical checksum found in the filesystem, '
                    'upgrade not needed.'
                )
                self.log(message)
```